docs/reference/migration/migrate_2_0.asciidoc

```diff
@@ -408,6 +408,9 @@ In addition some query builders have been removed or renamed:
 * `filtered(...)` removed. Use `filteredQuery(...)` instead.
 * `inQuery(...)` removed.
 
+`TermsLookupQueryBuilder` has been removed, please use `TermsQueryBuilder` with no values and set the corresponding
```

Comment:
I already replied to that comment explaining why the class is a pure delegate, that is only the case in our branch. This class should simply stay as-is, here and in master, please.